h/schemas/annotation.py

```diff
@@ -320,6 +323,13 @@ def _target_selectors(targets):
     return selectors
 
 
+def _target_description(targets) -> str | None:
+    if targets:
+        return targets[0].get("description", None)
+
+    return None
```

Comment:
If there's no `target[0].description` in the request body then the `annotation.target_description` DB column will end up containing `NULL`.